v2/pkg/protocols/http/request.go

```diff
@@ -361,6 +362,18 @@ func (request *Request) executeRequest(reqURL string, generatedRequest *generate
 				return errStopExecution
 			}
 		}
+		if request.options.Options.Debug || request.options.Options.DebugCorrelations {
```

Comment:
This debugging info should either be part of protocols/common/interactsh or it should also be added to network protocol since those also use oob functionality. I guess a generic solution would be much better

Reply:
Indeed - since i mainly use http i forgot about the other types. Will change and checkin later. Thanks for the feedback.